rest-service/manager_rest/resources_v2.py

```diff
@@ -18,13 +18,18 @@
 
 from manager_rest import resources
 from manager_rest.resources import (marshal_with,
-                                    exceptions_handled)
+                                    exceptions_handled,
+                                    verify_json_content_type,
+                                    verify_parameter_in_request_body)
 from manager_rest.resources import verify_and_convert_bool
 from manager_rest import models
+from manager_rest import responses
```

Comment:
Instead of importing `responses`, you should follow the current convention of importing `ProviderContext` in `responses_v2` and use it. If you disagree with this convention, then all other places should follow the same pattern.